### DISCUSSION TITLE
Tutorials Wanted!

### DISCUSSION THREAD

**Discussion by nothingismagick** [tauri-apps/tauri#1796] [Ideas]
Please help us know more about the tutorials / how-to's that would help you better understand how tauri works. Write a brief suggestion that explains what you want to learn.

**Comment by andremoreirafocus** [comment]
I am developing an Electron multiwindow app and I would like the great features regarding performance, security and small app size features  of Tauri, just to name the main ones. Could you please provide a tutorial to develop a multiwindow Tauri app? Thanks in advance!

**Reply by dibadibadiudiu** [reply]
Already, in the /tauri/examples/multiwindow/ directory。

**Comment by dibadibadiudiu** [comment]
I think we need more examples, which can be code fragments. When we can't understand the API, we can refer to the case implementation, and the case should be updated in time. I would also provide some examples.😁

**Comment by nothingismagick** [comment]
How to manage sidecars (including stopping them when Tauri exits).
See: https://github.com/tauri-apps/tauri/issues/1773

**Comment by onaxon** [comment]
Tauri may be the reason that I will decide to learn Rust.

I have experience in other languages but nothing in Rust. At this level, a very detailed tutorial describing creating an application that goes beyond the "Hello World", exploring the fundamental capabilities of Tauri, especially JS to Rust and Rust to JS communication, would be appreciated.

There is a Go project similar to Tauri called Wails, his [tutorial on the Todo App](https://wails.app/tutorials/todo/) is a good example.

**Comment by onaxon** [comment]
Also a tutorial of the creation of each [example app](https://github.com/tauri-apps/tauri/tree/dev/examples).
